actioncable/lib/action_cable.rb

```diff
@@ -37,6 +37,10 @@ module ActionCable
     },
     default_mount_path: '/cable'.freeze
   }
+  
+  # This protocol ought to be versioned separately
+  # from the rest of the Gem
+  PROTOCOLS = ["actioncable-protocol-1.0.0".freeze].freeze
```

Comment:
App-level protocol versioning is interesting. It could be helpful, at minimum, to provide a big hammer so apps can wiggle out of a stale-client compatibility mess by bumping the protocol.

For other cases, like changing channel behavior, I'd lean to leaving it up to apps to make changes in a compatible way. But, heck, leaning on bumping the app proto version could be simpler in many of these cases too.